internal/config/config.go

```diff
@@ -99,9 +125,10 @@ func LoadConfig(filePath string, fileName string) (*Config, error) {
 
 	// Config File
 	v.AddConfigPath(filePath)
+	v.AddConfigPath(".")
 	v.SetConfigName(fileName)
 
-	// If error reading file or file not found, use flag/env variables
```

Comment:
I do not understand neither the old or the new comment 😅

Reply:
Fixed in a nit picking commit. Would you resolve this conversation?